wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceInfoViewModel.swift

```diff
@@ -23,129 +23,104 @@ import WireDataModel
 import WireSyncEngine
 
 protocol DeviceDetailsViewActions {
-    var isMLSEnabled: Bool { get }
-    var isE2eIdentityEnabled: Bool { get }
     var isSelfClient: Bool { get }
     var isProcessing: ((Bool) -> Void)? { get set }
 
-    func fetchCertificate() async -> E2eIdentityCertificate?
-    func fetchMLSThumbprint() async -> String?
+    func getCertificate() async -> E2eIdentityCertificate?
+    func enrollClient() async -> E2eIdentityCertificate?
```

Comment:
```suggestion
    func enrollCertificate() async -> E2eIdentityCertificate?
```

Reply:
Well, I guess we are enrolling client here isn't it?

Reply:
Yes, were are enrolling a client

Reply:
But isn't this the same operation as what `getCertificate()` already does?